libraries/SITL/SITL.cpp

```diff
@@ -228,7 +228,7 @@ const AP_Param::GroupInfo SIM::var_info[] = {
     // @Description: Runs the simulation at multiples of normal speed. Do not use if realtime physics, like RealFlight, is being used
     // @Range: 1 10
     // @User: Advanced    
-    AP_GROUPINFO("SPEEDUP",       52, SIM,  speedup, -1),
+    AP_GROUPINFO("SPEEDUP",       52, SIM,  speedup, 1),
```

Comment:
This -1 is used elsewhere in the code as a flag value, eg.
```
    if (!is_equal(last_speedup, float(sitl->speedup)) && sitl->speedup > 0) {
```

maybe better to just use the method in the logger library and leave this thing sleep...
```suggestion
    AP_GROUPINFO("SPEEDUP",       52, SIM,  speedup, -1),
```